fishjam/media_events/server/server.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+package fishjam.media_events.server;
+
+import "fishjam/media_events/shared.proto";
+
+// Defines any type of message sent from Membrane RTC Engine to Peer
+message MediaEvent {
+  // SCHEMAS
+  message SimulcastConfig {
+    bool enabled = 1;
+    repeated media_events.Encoding active_encodings = 2;
+    repeated media_events.Encoding disabled_encodings = 3;
+  }
+
+  message Track {
+    string track_id = 1;
+    media_events.Metadata metadata = 2;
+    SimulcastConfig simulcast_config = 3;
+  }
+
+  message Endpoint {
+    string endpoint_id = 1;
+    string endpoint_type = 2;
+    media_events.Metadata metadata = 3;
+    repeated Track tracks = 4;
+  }
+
+  enum Status {
+    STATUS_UNSPECIFIED = 0;
+    STATUS_SILENCE = 1;
+    STATUS_SPEECH = 2;
+  }
```

Comment:
`Status` is a very generic term, and here we have declared it in the root scope. I would suggest renaming it to something like `VadStatus` or moving it to a different namespace, such as `VadNotification`